src/Namotion.Reflection/XmlDocsExtensions.cs

```diff
@@ -337,11 +337,11 @@ public static string GetXmlDocs(this ParameterInfo parameter, XmlDocsOptions? op
 
         /// <summary>Converts the given XML documentation <see cref="XElement"/> to text.</summary>
         /// <param name="element">The XML element.</param>
-        /// <param name="options">The XML docs reading and formatting options.</param>
+        /// <param name="formattingMode">The XML docs formatting options.</param>
         /// <returns>The text</returns>
-        public static string ToXmlDocsContent(this XElement? element, XmlDocsOptions? options = null)
+        public static string ToXmlDocsContent(this XElement? element, XmlDocsFormattingMode? formattingMode = null)
```

Comment:
please do not change the public API signature

Reply:
I did not realize it was public. I reverted that change so this pull request should be mergeable now. 😉